src/yea_wandb/backend.py

```diff
@@ -60,15 +60,33 @@ def _free_port(self, host):
     def start(self):
         if self._args.dryrun:
             return
-        if self._args.live:
-            return
         mockserver_bind = self._params["mockserver-bind"]
         mockserver_host = self._params["mockserver-host"]
         mockserver_relay = self._params["mockserver-relay"]
         mockserver_relay_remote_base_url = self._params[
             "mockserver-relay-remote-base-url"
         ]
         # TODO: consolidate with github.com/wandb/client:tests/conftest.py
+
+        # if we are using live mode. force mock_server plugin defaults
+        if self._args.live:
+            mockserver_bind = "0.0.0.0"
+            mockserver_host = "__auto__"
+            mockserver_relay = "true"
+            base_url = os.environ.get("WANDB_BASE_URL", "https://api.wandb.ai")
+            mockserver_relay_remote_base_url = base_url
+
+            # setup api key
+            api_key = os.environ.get("WANDB_API_KEY")
+            if not api_key:
+                auth = requests.utils.get_netrc_auth(base_url)
```

Comment:
this means that if users run locally it might use their netrc, are we ok with this?

Reply:
yes. it is by design.

say you are a developer and want to run a test. you want to use your currently configured api key.   Currently the user would have to do something super awkward like:
```
cat ~/.netrc | grep wandb.ai -A 2 | grep key | awk '{print $2}' 
```
or at least that is what i do :)